### PR TITLE
Flexbox

### PR DESCRIPTION
**Pull request by HansKallekleiv**
Should be reviewed together with https://github.com/equinor/webviz-core-components/pull/57.

### CODE REVIEW

**Comment by anders-kiaer** [comment]
Currently 🔴 from Travis. Update `webviz-config` requirement in `setup.py`.

File: webviz_subsurface/plugins/_inplace_volumes.py [hunk 1]
```diff
@@ -343,41 +322,37 @@ def plot_options_layout(self):
     @property
     def layout(self):
         """Main layout"""
-        return html.Div(
+        return wcc.FlexBox(
             id=self.ids("layout"),
             children=[
                 html.Div(
-                    style=self.style_layout,
+                    style={"flex": 5},
```
Comment (anders-kiaer) [inline webviz_subsurface/plugins/_inplace_volumes.py]: Did `width` work instead of `flex`?

**Comment by anders-kiaer** [comment]
Was not able to formally ask @HansKallekleiv for review, probably because it is your PR 🙂

**Comment by HansKallekleiv** [comment]
Added flexbox to "running time analysis".
Otherwise LGTM 👍

**Pull request merged**